blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/SSEEvents.java

```diff
@@ -0,0 +1,94 @@
+package io.jenkins.blueocean.blueocean_github_pipeline;
+
+import hudson.model.Item;
+import io.jenkins.blueocean.commons.ServiceException.UnexpectedErrorException;
+import jenkins.branch.MultiBranchProject;
+import jenkins.branch.OrganizationFolder;
+import jenkins.util.Timer;
+import org.jenkinsci.plugins.pubsub.MessageException;
+import org.jenkinsci.plugins.pubsub.PubsubBus;
+import org.jenkinsci.plugins.pubsub.SimpleMessage;
+
+import java.util.concurrent.TimeUnit;
+
+class SSEEvents {
```

Comment:
Perhaps you want to refactor `GithubPipelineCreateRequest` to use this instead of existing code to send SSE events?